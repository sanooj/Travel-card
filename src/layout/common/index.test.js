import React from 'react';
import { shallow } from 'enzyme';

import CommmonLayout from '.';
const title = 'Oyester card';
let wrapped = shallow(<CommmonLayout>{title}</CommmonLayout>);
describe('Layout', () => {
  it('should render the Title Component correctly', () => {   
    expect(wrapped).toMatchSnapshot();
  });
  it('renders the Header', () => { 
    expect(wrapped.find('h1').text()).toEqual('Oyester card');
  });
});