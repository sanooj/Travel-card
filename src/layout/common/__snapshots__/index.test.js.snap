// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render the Title Component correctly 1`] = `
<div
  className="min-h-full"
>
  <div
    className="bg-gray-100"
  >
    <div
      className="max-w-7xl mx-auto px-2 sm:px-6 lg:px-8"
    >
      <div
        className="relative flex items-center justify-between"
      >
        <h1
          className="my-3 text-center text-3xl font-extrabold text-gray-900"
        >
          Oyester card
        </h1>
      </div>
    </div>
  </div>
  <div
    className="max-w-7xl mx-auto px-2 py-12 sm:px-6 lg:px-8 "
  >
    Oyester card
  </div>
</div>
`;
